src/pyscal/utils/monotonicity.py

```diff
@@ -164,18 +164,17 @@ def clip_accumulate(
     Returns:
         np.array, copy of original.
     """
-    if isinstance(series, (list, np.ndarray)):
-        series = pd.Series(series, dtype="float64")
     if monotonicity["sign"] > 0:
         series = np.maximum.accumulate(series)
     else:
         series = np.minimum.accumulate(series)
+    series = np.array(series)
```

Comment:
I think this line is a no-op, since `np.maximum.accumulate` and `np.minimum.accumulte` always returns `np.ndarray`

Reply:
Good point 👍